compatibility/BUILD

```diff
@@ -51,6 +53,22 @@ platform_versions = [
     for platform_version in platform_versions
 ]
 
+[
+    daml_script_dar(sdk_version)
+    for sdk_version in sdk_versions
+]
+
+[
+    daml_script_test(sdk_version, platform_version)
```

Comment:
I think it would be helpful to use named arguments here to make it clear that `sdk_version` is mapped to `compiler_version` and `platform_version` to `runner_version`.

Reply:
Good idea, done.